### PR TITLE
Add Jummbox

### PR DESCRIPTION
**Pull request by Lihzahrd**
working

### CODE REVIEW

File: Jummbox/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "HiddenRealm",
+    "id": "444628005646499851"
+  },
+  "url": "jummbus.bitbucket.io/",
```
Comment (Bas950) [inline Jummbox/dist/metadata.json]: remove / at the end

File: Jummbox/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "HiddenRealm",
+    "id": "444628005646499851"
+  },
+  "url": "jummbus.bitbucket.io/",
+  "description": {
+    "english": "English"
+  },
```
Comment (Bas950) [inline Jummbox/dist/metadata.json]: Description should be:
"description": {
    "en": "DESCRIPTION OF WHAT THE SERVICE IS"
}

File: Jummbox/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "id": "444628005646499851"
+  },
+  "url": "jummbus.bitbucket.io/",
+  "description": {
+    "english": "English"
+  },
+  "service": "Jummbox",
+  "version": "1.0.0",
+  "logo": "https://jummbus.bitbucket.io/favicon.ico",
+  "thumbnail": "https://jummbus.bitbucket.io/favicon.ico",
```
Comment (Bas950) [inline Jummbox/dist/metadata.json]: Upload the images to a website like Imgur instead of using their own cdn.

File: Jummbox/presence.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+setInterval(10000, myOutsideHeavyLiftingFunction);
+//Run the function seperate from the UpdateData event every 10 seconds to get and set the variables which UpdateData picks up
+
+*/
+
+
+presence.on("UpdateData", async () => {
+    /*UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
```
Comment (Bas950) [inline Jummbox/presence.ts]: remove all comments as they are not needed.

File: Jummbox/dist/metadata.json (hunk 3 above)
Comment (Bas950) [reply Jummbox/dist/metadata.json]: The thumbnail should also be a screenshot of the website itself.

**Comment by Bas950** [comment]
You forgot to update the js version, it doesnt match the ts version.

**Comment by Bas950** [comment]
You must also upload proof of it working.

**Comment by Timeraa** [comment]
Maybe give the Pull Request a name?

**Pull request merged**